src/ethereum_clis/clis/ethereumjs.py

```diff
@@ -48,30 +49,47 @@ class EthereumJSExceptionMapper(ExceptionMapper):
     """Translate between EEST exceptions and error strings returned by EthereumJS."""
 
     mapping_substring: ClassVar[Dict[ExceptionBase, str]] = {
-        TransactionException.TYPE_4_TX_CONTRACT_CREATION: (
-            "set code transaction must not be a create transaction"
-        ),
-        TransactionException.INSUFFICIENT_ACCOUNT_FUNDS: (
-            "insufficient funds for gas * price + value"
-        ),
         TransactionException.TYPE_3_TX_MAX_BLOB_GAS_ALLOWANCE_EXCEEDED: (
             "would exceed maximum allowance"
         ),
         TransactionException.INSUFFICIENT_MAX_FEE_PER_BLOB_GAS: (
-            "max fee per blob gas less than block blob gas fee"
+            "Invalid 4844 transactions: undefined"
         ),
         TransactionException.INSUFFICIENT_MAX_FEE_PER_GAS: (
-            "max fee per gas less than block base fee"
+            "tx unable to pay base fee (EIP-1559 tx)"
+        ),
+        TransactionException.PRIORITY_GREATER_THAN_MAX_FEE_PER_GAS: (
+            "maxFeePerGas cannot be less than maxPriorityFeePerGas"
         ),
         TransactionException.TYPE_3_TX_PRE_FORK: (
             "blob tx used but field env.ExcessBlobGas missing"
         ),
-        TransactionException.TYPE_3_TX_INVALID_BLOB_VERSIONED_HASH: "has invalid hash version",
+        TransactionException.TYPE_3_TX_INVALID_BLOB_VERSIONED_HASH: (
+            "versioned hash does not start with KZG commitment version"
+        ),
         # This message is the same as TYPE_3_TX_MAX_BLOB_GAS_ALLOWANCE_EXCEEDED
         TransactionException.TYPE_3_TX_BLOB_COUNT_EXCEEDED: "exceed maximum allowance",
-        TransactionException.TYPE_3_TX_ZERO_BLOBS: "blob transaction missing blob hashes",
+        TransactionException.TYPE_3_TX_ZERO_BLOBS: "tx should contain at least one blob",
+        TransactionException.TYPE_3_TX_WITH_FULL_BLOBS: "Invalid EIP-4844 transaction",
+        TransactionException.TYPE_4_TX_CONTRACT_CREATION: (
+            'tx should have a "to" field and cannot be used to create contracts'
+        ),
+        TransactionException.TYPE_4_EMPTY_AUTHORIZATION_LIST: (
+            "Invalid EIP-7702 transaction: authorization list is empty"
+        ),
         TransactionException.INTRINSIC_GAS_TOO_LOW: "is lower than the minimum gas limit of",
-        TransactionException.INITCODE_SIZE_EXCEEDED: "max initcode size exceeded",
+        TransactionException.INITCODE_SIZE_EXCEEDED: (
+            "the initcode size of this transaction is too large"
+        ),
+        TransactionException.TYPE_4_TX_CONTRACT_CREATION: (
```

Comment:
This key is two times in this list? How does this work with mappings? What if the same keys have different values, does it throw? :thinking:

Reply:
It was a mistake, I only caught it when it started failing in hive.

Unfortunately python does not care about this:
```
>>> {'a': 'a', 'a': 'b'}
{'a': 'b'}
```
So we should be careful when defining these exceptions to not overwrite.

Reply:
I created a tracking issue for this because there should definitely be a way to disallow this.

Potentially create exception mapper instances instead of subclasses.